Update TAG Security & Privacy Questionnaire (#310)
w3c-fedid/digital-credentials: horizontal-reviews/security-privacy.md

> 01.  What information does this feature expose,
>      and for what purposes?

> Potentially high-value privacy sensitive claims from digital credentials, such as digital driver's licenses for both unique identification (such as for access to government services) and selective disclosure (such as for age verification). It does this through a one-time user-mediated communication channel from websites, to digital wallet applications, and back to websites. It is designed to be a better option than established lower-level communication channels like custom schemes, QR codes, and server-to-server network communication. The website can decide which information to request. The request can be inspected by the user agent and the specification recommends disclosing the requested information to the user in the permission prompt. The wallet application decides which information is actually shared based on the request.

Reviewer: ```suggestion
Potentially high-value, privacy-sensitive claims from digital credentials, such as both unique identification (such as for access to government services) and selective disclosures (such as for age verification) from digital driver's licenses. These requests are made from websites, to digital wallet applications, and back to websites, through user-mediated, one-time communication channels. These are designed to be better options than established lower-level communication channels like custom schemes, QR codes, and server-to-server network communications. A website can decide which information to request. A request can be inspected by the user agent, and the specification recommends that the information being requested be disclosed to the user in the permission prompt. The wallet application decides which information is actually shared, based upon the request.
```

> 02.  Do features in your specification expose the minimum amount of information
>      necessary to implement the intended functionality?

A primary use case of the digital credentials API is to enable users to selectively disclose only the minimum required information, such as a cryptographic attestation that the user holds a California driver’s license for an adult. The use of selective disclosure, however, is a decision for the verifier website based on the use case. There are legitimate scenarios, such as creating or recovering an account on a government website, where uniquely identifiable and potentially non-resettable personally identifiable information (PII) might be exposed.

> The API is designed to expect the use of response encryption so that this PII is exposed only to the requesting server, so that exposure to code running in the web page or browser can be avoided where possible. It is an [open question](https://github.com/WICG/digital-identities/issues/109) whether this response encryption is something we can reasonably enforce at this layer or not.

Reviewer: ```suggestion
The API is designed to expect the use of response encryption so that this PII is exposed only to the requesting server, such that exposure to code running in the web page or browser can be minimized or even avoided. It is an [open question](https://github.com/WICG/digital-identities/issues/109) whether this response encryption is something we can reasonably enforce at this layer.
```

> The API is also designed to require request transparency to enable user agents and operating systems to appropriately inform users about the level of privacy risk involved in the request. The specification has a set of recommendations ([1](https://www.w3.org/TR/digital-credentials/latest/#user-permission-and-transparency), [2](https://www.w3.org/TR/digital-credentials/latest/#mitigating-unnecessary-requests-for-government-credentials), [3](https://www.w3.org/TR/digital-credentials/latest/#mitigating-unnecessary-requests-for-non-government-credentials)) for implementers on how to appropriately inform users about the credentials exchange. For example, the [Chromium implementation](https://docs.google.com/document/d/1L68tmNXCQXucsCV8eS8CBd_F9FZ6TNwKNOaFkA8RfwI/edit) intends to show users stronger warnings in riskier scenarios such as those that lack the use of selective disclosure.

Reviewer: ```suggestion
The API is also designed to require request transparency to enable user agents and operating systems to appropriately inform users about the level of privacy risk involved in the request. The specification has a set of recommendations ([1](https://www.w3.org/TR/digital-credentials/latest/#user-permission-and-transparency), [2](https://www.w3.org/TR/digital-credentials/latest/#mitigating-unnecessary-requests-for-government-credentials), [3](https://www.w3.org/TR/digital-credentials/latest/#mitigating-unnecessary-requests-for-non-government-credentials)) for implementers, describing how to appropriately inform users about the credentials being exchanged. For example, the [Chromium implementation](https://docs.google.com/document/d/1L68tmNXCQXucsCV8eS8CBd_F9FZ6TNwKNOaFkA8RfwI/edit) intends to show users stronger warnings in riskier scenarios such as those that do not support selective disclosure.
```

> 15.  Does this specification have both "Security Considerations" and "Privacy
>      Considerations" sections?

> The section has [extensive Privacy Considerations](https://w3c-fedid.github.io/digital-credentials/#privacy-considerations), most which are still evolving as the group works through the many Privacy-related challenges of the digital credentials space.

Author: ```suggestion
The section has [extensive Privacy considerations](https://w3c-fedid.github.io/digital-credentials/#privacy-considerations), most which are still evolving as the group works through the many Privacy-related challenges of the digital credentials space.
```

> The specification has [extensive Privacy Considerations](https://w3c-fedid.github.io/digital-credentials/#privacy-considerations), most which are still evolving as the group works through the many Privacy-related challenges of the digital credentials space.

Author: ```suggestion
The specification has [extensive Privacy considerations](https://w3c-fedid.github.io/digital-credentials/#privacy-considerations), most which are still evolving as the group works through the many Privacy-related challenges of the digital credentials space.
```
